Plugin/CppApi/source/AbstractTool.cpp

```diff
@@ -18,39 +18,103 @@ namespace ArcGISRuntime
 {
 namespace Toolkit
 {
+/*!
+  \class Esri::ArcGISRuntime::Toolkit::AbstractTool
+  \inmodule ArcGISQtToolkit
+  \ingroup ToolCoordinateConversion
+  \ingroup ToolArcGISCompass
+  \brief Base class for classes that implement individual tools.
+  \since Esri::ArcGISRuntime 100.2
 
+  AbstractTool provides functions and properties that all derived tools share.
+ */
```

Comment:
Maybe add something like - "This is an abstract class: you cannot create instances of this type directly but instead create a derived tool type."